backend/src/test/java/codezap/tag/repository/TemplateTagJpaRepositoryTest.java

```diff
@@ -32,52 +33,114 @@ class TemplateTagJpaRepositoryTest {
+        member = memberRepository.save(new Member("user1@test.com", "pp", "salt1"));
+
+        category = categoryRepository.save(new Category("Category 1", member));
     }
 
     @Test
+    @DisplayName("findAllByTemplateTest 조회 성공")
+    void findAllByTemplateTest() {
+        //given
+        Template template = templateRepository.save(new Template(member, "testTemplate", "testTemplate", category));
+
+        Tag tag1 = tagRepository.save(new Tag("tag1"));
+        Tag tag2 = tagRepository.save(new Tag("tag2"));
+        tagRepository.save(new Tag("tag3"));
+
+        TemplateTag templateTag1 = templateTagRepository.save(new TemplateTag(template, tag1));
+        TemplateTag templateTag2 = templateTagRepository.save(new TemplateTag(template, tag2));
+
+        //when
+        List<TemplateTag> templateTags = templateTagRepository.findAllByTemplate(template);
+
+        //then
+        assertThat(templateTags).containsExactly(templateTag1, templateTag2);
+    }
+
+    @Test
+    @DisplayName("findDistinctByTemplateIn 조회 테스트")
     void testFindDistinctByTemplateIn() {
         // given
-        List<Long> templateIds = List.of(1L, 2L, 3L);
+        Template template1 = templateRepository.save(
+                new Template(member, "testTemplate1", "testTemplate1", category));
+        Template template2 = templateRepository.save(
+                new Template(member, "testTemplate2", "testTemplate2", category));
+        Template template3 = templateRepository.save(
+                new Template(member, "testTemplate3", "testTemplate3", category));
+
+        Tag tag1 = tagRepository.save(new Tag("tag1"));
+        Tag tag2 = tagRepository.save(new Tag("tag2"));
+        Tag tag3 = tagRepository.save(new Tag("tag3"));
+
+        templateTagRepository.save(new TemplateTag(template1, tag1));
+        templateTagRepository.save(new TemplateTag(template3, tag1));
+        templateTagRepository.save(new TemplateTag(template3, tag3));
 
         // when
-        List<Long> result = templateTagRepository.findDistinctByTemplateIn(templateIds);
+        List<Long> result = templateTagRepository.findDistinctByTemplateIn(
+                List.of(template1.getId(), template3.getId())
+        );
 
         // then
-        assertThat(result).isNotEmpty();
-        assertThat(result.stream().distinct().count())
-                .isEqualTo(result.size());
+        assertThat(result).containsExactly(tag1.getId(), tag3.getId())
+                .hasSize(2);
+    }
+
+    @Test
+    @DisplayName("findAllTemplateIdInTagIds 조회 테스트")
+    void findAllTemplateIdInTagIds() {
+        //given
+        Template template1 = templateRepository.save(new Template(member, "title1", "description1", category));
+        Template template2 = templateRepository.save(new Template(member, "title2", "description2", category));
+        Template template3 = templateRepository.save(new Template(member, "title3", "description3", category));
```

Comment:
```suggestion
        Template template1 = templateRepository.save(
                new Template(member, "title1", "description1", category));
        Template template2 = templateRepository.save(
                new Template(member, "title2", "description2", category));
        Template template3 = templateRepository.save(
                new Template(member, "title3", "description3", category));
```

위에서 한 것처럼 개행하면 어떨까요?